docs/IntegrationWithExistingApps.md

```diff
@@ -83,12 +83,16 @@ The keys to integrating React Native components into your Android application ar
 
 The [Android Getting Started guide](/react-native/docs/getting-started.html) will install the appropriate prerequisites (e.g., `npm`) for React Native on the Android target platform and your chosen development environment.
 
+> To going everything straightforward, make sure your android project is under `$root/android`.
```

Comment:
@JoelMarcey, agree, I am not a native english speaker. Thanks for the feedback. I will update the PR.